charts/remoting-agent/values.yaml

```diff
@@ -0,0 +1,239 @@
+    add: 
+      - NET_BIND_SERVICE
+  # -- Whether the root filesystem is mounted in read-only mode
+  readOnlyRootFilesystem: true
+  # -- Controls [Privilege Escalation](https://kubernetes.io/docs/concepts/security/pod-security-policy/#privilege-escalation) enabling setuid binaries changing the effective user ID
+  allowPrivilegeEscalation: false
+  # -- Requires the container to run without root privileges
+  runAsNonRoot: true
+  # -- The container's process will run with the specified uid
+  runAsUser: 10001
+  # -- Processes within a pod will belong to this guid
+  runAsGroup: 30000
+
+livenessProbe:
+  # -- Whether to enable kubernetes [liveness-probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+  enabled: true
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+  # -- Number of seconds each period lasts.  
+  periodSeconds: 60
+  # -- number of successful tries which reenables liveness
+  successThreshold: 1
+  # -- number of seconds until a timeout is assumed
+  timeoutSeconds: 5
+
+readinessProbe:
+  # -- Whether to enable kubernetes readiness-probes
+  enabled: true
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 3
+  # -- Number of seconds each period lasts.  
+  periodSeconds: 300
+  # -- number of successful tries which reenables liveness
+  successThreshold: 1
+  # -- number of seconds until a timeout is assumed
+  timeoutSeconds: 5
+
+startupProbe:
+  # -- Whether to enable kubernetes startup-probes
+  enabled: true
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded
+  failureThreshold: 18
+  # -- Number of seconds after the container has started before liveness probes are initiated.
+  initialDelaySeconds: 60
+  # -- Number of seconds each period lasts.  
+  periodSeconds: 30
+  # -- number of successful tries which reenables liveness
+  successThreshold: 1
+  # -- number of seconds until a timeout is assumed
+  timeoutSeconds: 5
+
+## Endpoints exposed by the remoting agent
+agent:
+  endpoints:
+    ## Default api exposing health checks etc
+    default:
+      # -- The network port, which the "default" api is going to be exposed by the container, pod and service
+      port: "8081"
+      # -- An auth object for default security
+      auth: {}
+      # -- The path mapping the "default" api is going to be exposed by
+      path: ""
+      # -- The path mapping the "default" api is going to be exposed by
+      regex: /(.*)
+
+service:
+  # -- [Service type](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) to expose the running application on a set of Pods as a network service.
+  type: ClusterIP
+
+## Ingress declaration to expose the network service.
+ingresses:
+  ## Public / Internet facing Ingress
+  - enabled: true
```

Comment:
Default value to ingress should be `enabled: false`. Please refer to [TRG 5.01](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-01). 

Also there should be no ingress-nginx related annotations as one could use a different ingress controller. Maybe just comment out and add an appropriate comment when to use these annotations.

Reply:
done